packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -58,14 +58,16 @@ const StyledLink = styled.a`
   }
 
   @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    ${({ script }) => script && getGreatPrimer(script)};
+    ${({ script, typography }) =>
+      script && typography !== 'pica' && getGreatPrimer(script)}
```

Comment:
In the case where you have typography set to 'pica', you keep the media min-width block with no content. It'd be better to not have this block.